### PR TITLE
Drawing a Line Chart

### PR DESCRIPTION
**Pull request by SideswipeN7**

### CODE REVIEW

File: stretching/MainWindow.xaml.cs [hunk 1]
```diff
@@ -9,12 +14,40 @@ namespace Stretching
...
+
         public MainWindow()
         {
             InitializeComponent();
+  
             app_ = new StretchApp(this);
+            
+
+
+            CartesianChart1.Series = new SeriesCollection
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs]: Don't draw chart here do it in app as part of solver

File: stretching/App/StretchApp.cs [hunk 2]
```diff
@@ -143,6 +143,9 @@ private void OnDataLoad()
         private void PlotGraph()
         {
             //TODO: Plot graph
+
+
+
             throw new NotImplementedException();
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs]: Here You should draw raph

File: stretching/Stretching.csproj [hunk 3]
```diff
@@ -35,8 +35,17 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="LiveCharts, Version=0.9.7.0, Culture=neutral, PublicKeyToken=0bc1f845d1ebb8df, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>bin\Debug\LiveCharts.dll</HintPath>
+    </Reference>
+    <Reference Include="LiveCharts.Wpf, Version=0.9.7.0, Culture=neutral, PublicKeyToken=0bc1f845d1ebb8df, processorArchitecture=MSIL">
```
Comment (SideswipeN7) [inline stretching/Stretching.csproj:42]: Also install this via nugget so it can be seen in packages.config

**Review by SideswipeN7 — COMMENTED** [review]
Rework
-change packages.config
-don't draw in main window
-draw gaph in app correct method
-use data from parser to draw graph

File: stretching/App/StretchApp.cs (hunk 2 above)
Comment (Ossj400) [reply stretching/App/StretchApp.cs]: I know where :D I just put it there, coz I didnt know how to implement this here, and how to use parser data..

File: stretching/App/StretchApp.cs [hunk 4]
```diff
@@ -31,6 +31,8 @@ class StretchApp
         public StretchApp(MainWindow window) : this()
         {
             window_ = window;
+            PlotGraph();
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs]: Wrong place

File: stretching/App/StretchApp.cs [hunk 5]
```diff
@@ -57,6 +59,7 @@ private StretchApp()
          */
         public void ReadFile()
         {
+
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs:64]: Remove this

File: stretching/App/StretchApp.cs [hunk 6]
```diff
@@ -134,6 +137,7 @@ private void OnDataLoad()
             else
             {
                 //TODO: Show message cant plot graph
+
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs:142]: remove this

File: stretching/App/StretchApp.cs [hunk 7]
```diff
@@ -31,6 +31,8 @@ class StretchApp
         public StretchApp(MainWindow window) : this()
         {
             window_ = window;
+            PlotGraph();
+
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs]: remove this

File: stretching/App/StretchApp.cs [hunk 8]
```diff
@@ -142,8 +146,22 @@ private void OnDataLoad()
          */
         private void PlotGraph()
         {
-            //TODO: Plot graph
-            throw new NotImplementedException();
+            window_.CartesianChart1.Series = new LiveCharts.SeriesCollection
```
Comment (SideswipeN7) [inline stretching/App/StretchApp.cs]: Wrong implementation

File: stretching/MainWindow.xaml.cs [hunk 9]
```diff
@@ -9,12 +14,17 @@ namespace Stretching
...
         StretchApp app_;
+        private string[] Labels;
+
         public MainWindow()
         {
             InitializeComponent();
             app_ = new StretchApp(this);
         }
 
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:24]: remove this

File: stretching/MainWindow.xaml.cs [hunk 10]
```diff
@@ -9,12 +14,17 @@ namespace Stretching
...
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:25]: remove this

File: stretching/MainWindow.xaml.cs [hunk 11]
```diff
@@ -9,12 +14,17 @@ namespace Stretching
...
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:26]: remove this

File: stretching/MainWindow.xaml.cs [hunk 12]
```diff
@@ -9,12 +14,17 @@ namespace Stretching
     public partial class MainWindow : Window
     {
         StretchApp app_;
+        private string[] Labels;
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:17]: remove this

File: stretching/MainWindow.xaml.cs [hunk 13]
```diff
@@ -24,5 +34,7 @@ private void Save_Click(object sender, RoutedEventArgs e)
         {
             app_.SaveGraph();
         }
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:36]: remove this

File: stretching/MainWindow.xaml.cs [hunk 14]
```diff
@@ -24,5 +34,7 @@ private void Save_Click(object sender, RoutedEventArgs e)
...
+
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs:37]: remove this

File: stretching/MainWindow.xaml.cs [hunk 15]
```diff
@@ -9,12 +14,17 @@ namespace Stretching
     public partial class MainWindow : Window
     {
         StretchApp app_;
+        private string[] Labels;
```
Comment (SideswipeN7) [inline stretching/MainWindow.xaml.cs]: remove this

**Pull request merged**